markdown/setup/squirrel-driver-setup.md

```diff
@@ -54,5 +54,3 @@
 
     ![Example](../images/squirrel/squirrel7.png)
```

Comment:
is "squirrel7.png" an existing file or a new file?

Reply:
It is a new file include in this PR. GitHub may have rendering issues since the PR is from a forked repository, please check [the source file](https://github.com/awslabs/amazon-timestream-driver-jdbc/blob/9db9f00c53e5ddff48d8b161f457a1a12dc1a34b/markdown/setup/squirrel-driver-setup.md) to see pictures.